verilog/analysis/checkers/BUILD

```diff
@@ -1627,6 +1628,45 @@ cc_test(
     ],
 )
 
+cc_library(
+    name = "parameter_name_prefix_rule",
+    srcs = ["parameter_name_prefix_rule.cc"],
+    hdrs = ["parameter_name_prefix_rule.h"],
+    deps = [
+        "//common/analysis:lint_rule_status",
+        "//common/analysis:syntax_tree_lint_rule",
+        "//common/analysis/matcher",
+        "//common/analysis/matcher:bound_symbol_manager",
+        "//common/strings:naming_utils",
+        "//common/text:config_utils",
+        "//common/text:symbol",
+        "//common/text:syntax_tree_context",
+        "//common/text:token_info",
+        "//verilog/CST:parameters",
+        "//verilog/CST:verilog_matchers",
+        "//verilog/analysis:descriptions",
+        "//verilog/analysis:lint_rule_registry",
+        "//verilog/parser:verilog_token_enum",
+        "@com_google_absl//absl/strings",
```

Comment:
this will probably not work without internet, however I guess we can fix it when we'll get to that bridge